src/app/form-entry/control-calculators/can-calculate.ts

```diff
@@ -0,0 +1,8 @@
+import { Observable } from 'rxjs/Rx';
+
+export interface CanCalculate {
+  valueChanges?: Observable<any>;
```

Comment:
We no-longer need the 'valueChanges' member in a CanCalculate form control interface